src/prefs/ImportExportPrefs.cpp

```diff
@@ -109,14 +128,40 @@ void ImportExportPrefs::PopulateOrExchange(ShuttleGui & S)
       [&](const PopulatorItem &item, auto &) { item.mPopulator(S); },
       &top, &PopulatorItem::Registry());
 
+
+   auto musicImportsBox = S.StartStatic(XO("Music Imports"));
+   {
+      const auto header = S.AddVariableText(
+         XO("When Audacity detects music in file imported on empty project"));
+#if wxUSE_ACCESSIBILITY
+      if (musicImportsBox != nullptr)
+      {
+         musicImportsBox->SetName(header->GetLabel());
+         safenew WindowAccessible(musicImportsBox);
+      }
+#endif
+      S.StartPanel();
+      {
+         S.StartRadioButtonGroup(ImportExportPrefs::MusicFileImportSetting);
+         {
+            S.TieRadioButton();
+            S.TieRadioButton();
+            S.TieRadioButton();
+         }
+         S.EndRadioButtonGroup();
+      }
+      S.EndPanel();
+   }
+   S.EndStatic();
```

Comment:
OK so this commit corrects the previous, using ShuttleGui.

I wonder if a `PopulatorItem` could be used and other tricks to make the music information retrieval a non-intrusively defined module.  Also breaking the dependencies added in ProjectFileManager.cpp.

Reply:
?